recipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "k-means-constrained" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3cf1601ebe91c2317fd595399153e81c450085b15a072ea67f9850175a12102
+
+build:
+  # or-tools is missing on Windows due to OpenBLAS requirement 
+  skip: true  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython 0.29.33
+    - numpy 1.23.5
+    - pip
+  run:
+    - python
+    - ortools-python >=9.4
+    - scipy >=1.6.3
```

Comment:
Where does 1.6.3 come from? They seem ot be using >=0.13.3 upstream (https://github.com/joshlk/k-means-constrained/blob/v0.5.1/requirements.txt).

Reply:
https://github.com/joshlk/k-means-constrained/blob/master/requirements.txt#L2

Sorry for the 0.5.1 confusion 😞